server/test/views/BaseHtmlLayoutTest.java

```diff
@@ -26,7 +26,8 @@ public void addsDefaultContent() {
     assertThat(content.body()).contains("<!DOCTYPE html><html lang=\"en\">");
 
     assertThat(content.body())
-        .containsPattern("<link href=\"/assets/stylesheets/tailwind.css\" rel=\"stylesheet\">");
+        .containsPattern(
+            "<link href=\"/assets/stylesheets/[a-z0-9]+-tailwind.css\" rel=\"stylesheet\">");
```

Comment:
Out of curiosity, how are the changes to this file relevant? They seem unrelated.

Reply:
Without this the unit tests were failing..

Reply:
That's interesting. That seems to be related to my change of enabling fingerprinting for all files. I submitted it earlier this week. And I did expect tailwind.css to have fingerprint as well but for some reason the test didn't fail so I didn't update it. Hm, you reverted this change because it's no longer failing?

Reply:
Yes , it worked fine in CI even after reverting it.. so reverted it.